mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -54,27 +68,48 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	METHOD method_1013 isValid ()Z
 	METHOD method_1014 expand (D)Lnet/minecraft/class_238;
 		ARG 1 value
-	METHOD method_17939 getXSize ()D
-	METHOD method_17940 getYSize ()D
-	METHOD method_17941 getZSize ()D
+	METHOD method_17939 getXLength ()D
+	METHOD method_17940 getYLength ()D
+	METHOD method_17941 getZLength ()D
```

Comment:
Why change this? Every other size getter uses `getSizeX`/`Y`/`Z`. Using `length` in this one would be weird.

Reply:
This change was originally a suggestion of mine, but I agree that we should probably stick to established conventions.

Reply:
`getSizeX` IMO, is counter to the CONVENTIONS.md

> To make code as easy to read as possible, keep names in the natural language order. For example, a class representing a chest block entity should be named `ChestBlockEntity` rather than `BlockEntityChest`.

Reply:
What? `getXSize` breaks the convention. There's no such thing as an "x size" (unless you're talking about the size of the "x" character in a font). There is though a "size's x", the x component of the size.

Also, the conventions mention this explicitly: https://github.com/FabricMC/yarn/blob/1.14.4/CONVENTIONS.md#coordinates

Reply:
Get X Size = get the size in the X dimension
Get X Length = get the length of the X component

Compare that to a real conversaion:

"What is your cup('s) size?"
"My cup('s) size is DD"

As opposed to:

"What is your size('s) cup?"
"My size has a cup?"

Do you even english?

Reply:
You can also consider them a mutation of `x`, `y`, and `z`.

Where each component is a scalar _range_. If the mins and maxes of each dimension where refactored into a dedicated `Range` object the equivalent of these methods would all be:
`getZ().length()`, `getY().length()`, `getZ().length()`.

Or.

`getZ().getSize()`, `getY().getSize()`, `getZ().getSize()`.

Reply:
I have never heard anyone refer to the size in the x dimension as the "x size". 

> What is your size('s) cup?

A size doesn't have a cup. But a cup has a size, and that size has three components.

A size doesn't have a box. But a box has a size, and that size has three components.

> Do you even english?

Do you even logic?

Also, ironically, you forgot the capital letter in English exactly in the sentence you insult my English.

> Where each component is a scalar range. If the mins and maxes of each dimension where refactored into a dedicated Range object the equivalent of these methods would all be:
getX().length(), getY().length(), getZ().length().

I would never guess that a method named `getX` returns a range, I'd instead assume it returns the position on the x axis. If I saw `getX().length()` I'd be confused about how a position can possibly have a length, just as I'd be confused when I see `getXLength`.

If you really want to use the (weird) range naming, then it would have to be `getXRangeLength` (get the x [coordinate's] range length) to be clear.

Also, by the same argument as yours, I can argue that `getSizeX` is correct, because it represents `getSize().getX()`. It's obviously more common to see methods that return sizes than methods that return ranges, so this would be the better option even if the other one was correct (not saying it is).

I suggest that this PR doesn't get merged as it is until a PR changing `CONVENTIONS.MD` is merged, since it's currently breaking the conventions.

Reply:
I'll agree to that until someone else can weigh in on this.

As it is, I'm not against renaming the methods to something else. Just as it stands, though the pre-X version are _obviously_ better in my opnion, neither options `getSizeX`, `getLengthX`, `getXSize` or `getXLength` are very elegant names due to the mixing of "dimension-axis + arbitrary-modifier".